rtsp-to-web/rootfs/data/config.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think that you cannot store data inside of the image at /data because it will be mounted as volume on top of it. You need instead to create it during the initialization at /data.

Reply:
Thanks for catching this. This is now updated to copy from /app/config.json to /data/config.json if the file does not already exist. Maybe that works?